libtcmu_log.h

```diff
@@ -14,6 +14,7 @@
 #include <stdarg.h>
 #include <stdbool.h>
 
+#define TCMU_LOG_CRIT	LOG_CRIT
```

Comment:
just throw a comment at the end, to look alike others

Reply:
Sure, will do it.
Thanks